### DISCUSSION TITLE
Current user [Apr 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5594] [Discord Threads]
> 🙋‍♂️ **lcslouis**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### <@371027803321139210> Not possible at the moment. What's your use case?  
#  
> 🙋‍♂️ **lcslouis**  
#### A logged in user submitting a form. When form is submitted that record could show who submitted it and who owns that record.  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### ok. will add this as a feature request to github  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### https://github.com/nocodb/nocodb/issues/5378  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1090131782353027203)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
